taskcluster/scripts/pipeline/train-taskcluster.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
question: Should we be supporting pretrained models for student training as well? (This is probably a question for @eu9ene).

(If we should, I suggest this is a future enhancement, and not something for this PR.)

Reply:
Are we handling the spot terminations gracefully in Taskcluster now? Last time I tried to continue from a preempted VM I couldn't because the worker didn't fail gracefully and lost the checkpoint artifacts.